src/main/java/org/fundacionjala/sfdc/util/CommonActions.java

```diff
@@ -0,0 +1,21 @@
+package org.fundacionjala.sfdc.util;
+
+import org.fundacionjala.sfdc.commons.DriverManager;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.openqa.selenium.support.ui.WebDriverWait;
+
+public class CommonActions {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Add a private constructor to hide the implicit public one. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1118)